.travis.yml

```diff
@@ -92,6 +95,9 @@ script:
       -e ARROW_PARQUET=${ARROW_PARQUET:-ON} \
       -e PARQUET_BUILD_EXAMPLES=${PARQUET_BUILD_EXAMPLES:-ON} \
       -e PARQUET_BUILD_EXECUTABLES=${PARQUET_BUILD_EXECUTABLES:-ON} \
+      -e cares_SOURCE=${cares_SOURCE:-AUTO} \
+      -e gRPC_SOURCE=${gRPC_SOURCE:-AUTO} \
+      -e Protobuf_SOURCE=${Protobuf_SOURCE:-AUTO} \
```

Comment:
Could you sort this list alphabetically?

Reply:
Could you use empty string (`${XXX:-}`) instead of `AUTO` as the default value?
Because empty string is the default value in `ci/script/cpp_build.sh`.

Reply:
Why aren't we simply using `ci/script/cpp_build.sh`?

Reply:
Sorry, I have not understood the high-level view of Arrow CI yet. I followed the existing script.   
Is your comment to suggest to run tests without docker by directly executing `cpp_build.sh`?

@kou would it be possible to help me if I am wrong.

Reply:
We're already using `ci/script/cpp_build.sh` in Docker container.